services/newrelic/newrelic.ini

```diff
@@ -24,11 +24,6 @@
 
 [newrelic]
 
-# You must specify the license key associated with your New
-# Relic account. This key binds the Python Agent's data to your
-# account in the New Relic service.
-license_key = 889221e414af81d4877cc57b9bfd9413ac122dfe
```

Comment:
Removing the license key is awesome. But I can't tell from the commit message what you replaced it with? Was it the env variable NEW_RELIC_LICENSE_KEY? And, if so, how are you currently distributing it out?